src/components/scenes/Loans/ManageCollateralScene.js

```diff
@@ -91,20 +96,21 @@ export const ManageCollateralScene = <T: $Keys<ParamList>>(props: Props<T>) => {
   const wallets = useWatch(account, 'currencyWallets')
 
   // Skip directly to LoanStatusScene if an action for the same actionOpType is already being processed
-  const existingProgramId = useRunningActionQueueId(actionOpType, currencyWallet.id)
+  const existingProgramId = useRunningActionQueueId(actionOpType, borrowEngineWallet.id)
   if (existingProgramId != null) navigation.navigate('loanStatus', { actionQueueId: existingProgramId })
 
   // Flip input selected wallet
-  const [selectedWallet, setSelectedWallet] = useState<EdgeCurrencyWallet>(currencyWallet)
+  const [selectedWallet, setSelectedWallet] = useState<EdgeCurrencyWallet>(borrowEngineWallet)
   const [selectedTokenId, setSelectedTokenId] = useState<string | void>(defaultTokenId)
   const selectedWalletName = useWatch(selectedWallet, 'name') ?? ''
-  const { currencyCode: selectedWalletCurrencyCode } = selectedTokenId == null ? currencyInfo : allTokens[selectedTokenId]
+  const { currencyCode: selectedCurrencyCode } = selectedTokenId == null ? currencyInfo : allTokens[selectedTokenId]
```

Comment:
For the future, whenever you have a lot of renames in a PR, it helps to rebase the rename commits before the the actual logic change commit so review is faster. I try to do this, but I don't always do this either, but it's still best practice. Sometimes I avoid renames just for the sake of clearer commits and faster PR turn-around (even though the rename is totally needed).